src/ensembl/src/services/indexeddb-service.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { openDB, IDBPDatabase } from 'idb';
+
+const DB_NAME = 'ensembl-website';
+const DB_VERSION = 1;
```

Comment:
Do we need a separate file/config to store these?

Reply:
Not sure. Baby steps.